packages/fuel-indexer/src/commands/run.rs

```diff
@@ -67,6 +67,10 @@ pub async fn exec(args: IndexerArgs) -> anyhow::Result<()> {
         Manifest::from_file(&p).unwrap()
     }) {
         Some(m) => {
+            info!(
```

Comment:
We shouldn't make the link dependent on whether the user included a manifest or not, we should do it regardless (maybe just above this line)

Reply:
@ra0x3 I've added emojis, and now show two different urls depending on the match arm. This line seems to be the earliest point in the program that I have access to the manifest values (without adding more logic to parse the PathBuf and read the file, which seemed unnecessary).

`exec` is first function  called after `run_cli()`. Let me know your thoughts.